apps/ui/fly.toml

```diff
@@ -1,4 +1,4 @@
-app = "promptu"
+app = "promptu-isatest12"
```

Comment:
The PR doesn't include changes in the doc, only changes in the toml. You can raise a PR directly in the `make-or-break-it` org btw

Reply:
lol, yap got mixed here.